pypeman/nodes.py

```diff
@@ -247,8 +247,15 @@ def process(self, msg):
 
 class Drop(BaseNode):
```

Comment:
English:
This node can be used
This node is used
Use this node

"this node used to" means that before the node had the mentioned functionality, but it has it no more.

Reply:
Fixed